packages/apps/kadena-docs/src/utils/hasSameBasePath.ts

```diff
@@ -2,8 +2,13 @@ export const hasSameBasePath = (
   pathname1?: string,
   pathname2?: string,
 ): boolean => {
+  let basePathLength = 3;
+  if (process.env.NEXT_PUBLIC_APP_DEV === 'test') {
```

Comment:
Is this just for very explicit testing, or can we use `NODE_ENV` here?

Reply:
in Next the NODE_ENV is set explicitly by Next itself